src/main/java/Event.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Represents event task.
+ */
+public class Event extends Task {
+    String by;
```

Comment:
the variable might benefit from having a more detailed variable name eg. such as time!

Reply:
overall, clean code that mostly follows style guide!